samples/rest/embeddings.sh

```diff
@@ -0,0 +1,32 @@
+set -eu
+
+echo "[START embed]"
+# [START embed]
```

Comment:
Make sure to use the same tags as in the sheet:

ebed_content and batch_embed_contents